lib/chars.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+angular.module( 'lnFilters' )
+  .filter( 'chars', chars );
+
+function chars() {
+  return function ( input, chars, breakOnWord ) {
+    if ( isNaN( chars ) ) return input;
+    if ( chars <= 0 ) return '';
+    if ( input && input.length > chars ) {
+      input = input.substring( 0, chars );
```

Comment:
Can we use `trim()` before to remove any extra space that might exist on the original input.